tox.ini

```diff
@@ -17,51 +17,50 @@ whitelist_externals =
 
 [testenv:es]
 commands =
-    make -f Makefile-docker install_python_test_dependencies
-    pytest -m "es_tests and not needs_locales_compilation and not static_assets" --ignore=tests/ui/ -v src/olympia/{posargs}
+    make -f Makefile-docker update_deps
+    pytest -m "es_tests and not needs_locales_compilation" --ignore=tests/ui/ -v {posargs}
```

Comment:
Please don't change this, looks like a merge gone bad? The same for all the other environments in the `tox.ini`. We don't want to run `update_deps` directly here for performance reasons.